setup.py

```diff
@@ -34,7 +34,9 @@ def set_gcc():
 
     # For macports and homebrew
     patterns = ['/opt/local/bin/gcc-mp-[0-9].[0-9]',
-                '/usr/local/bin/gcc-[0-9].[0-9]']
+                '/opt/local/bin/gcc-[0-9]',
```

Comment:
Should this be `/opt/local/bin/gcc-mp-[0-9]`? and the next line `/usr/local/bin/gcc-[0.9.[0-9]`? That's because `gcc-mp` is the macports version and installs into `/opt/local/`, and the Homebrew version goes into `/usr/local/` and does not have the `mp` infix?